docs/sass_usage.md

```diff
@@ -0,0 +1,68 @@
+## Introduction and overview of sass
+
+sass (syntactically awesome style sheets) is a scripting language that extends css. sass brings typical progamming features and functionality such as functions,loops,maxims and inheritance into css thereby making css more organised, easier to mainntain and create complex styles.It is often refered to as **css with super powers**. sass has the two syntax that is the .scss and the .sass. The .scss is popularlily used because of it's simple syntax that is similar to css. sass can be used to build css frameworks and libraries.
+
+# Prerequisites
+
+Before being able to use sass, make sure to have the following installed
+
+- [nvm (Node Version Manager)](https://github.com/nvm-sh/nvm): To manage Node.js versions.
+
+- [node.js] (https://nodejs.org/en/):javascript runtime enviroment, the foundation on which sass is built.
+
+- [npm] (https://www.npmjs.com/): Package manager for node.js
+
+- [gulp] (https://gulpjs.com/docs/en/getting-started/quick-start/):Javasript toolkit that helps automate the running of task.It can be useed to watch a .scss file and compile it to a .css file automatically.  **note** gulp is optional here as other tools can be used for example visualsstudio code has an extention for sass. sass itself has commands that can be run on the terminal to compile from .scss to .css.
+
+- [sass] (https://sass-lang.com/install/) :The preprocessor programming language that is either interpreted or compiled into css.
+
+# installation
```

Comment:
You should probably also update your doc, because this is very general. In our case, we're using `vite` & `react`